src/middlewared/middlewared/plugins/jail.py

```diff
@@ -343,6 +344,14 @@ def common_validation(self, verrors, options, update=False, jail=None):
                         'Please Enter two valid and different '
                         f'space/comma-delimited MAC addresses for {key}.'
                     )
+        if options['uuid']:
+            valid = True if re.match(r"^[a-zA-Z0-9\._-]+$", name) else False
+
+            if not valid:
+                verrors.add(
+                    'options.uuid',
+                    'Invalid character in {options["uuid"]}, please remove it.
```

Comment:
F strings ;)

Reply:
What is the invalid character? Or it should say what are the valid characters, perhaps?